src/main/webapp/WEB-INF/tags/shared/commentsPanel.tag

```diff
@@ -19,10 +19,15 @@
         <c:set var="bodyTitle" value="Comments for students" />
     </c:otherwise>
 </c:choose>
-<div class="panel panel-primary">
-    <div class="panel-heading">
+<div class="panel panel-primary" data-toggle="collapse" data-target="#panelBodyCollapse">
+    <div class="panel-heading cursor-pointer"
```

Comment:
@tarun14110 my apologies, I had overlooked this, the data-toggle and data-target attributes should be in the `panel-heading` element rather than the `.panel` element.